code/kris/html_css_flask/lab-02/templates/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="{{url_for('static', filename = 'styles.css')}}">
+
+
+    <title>Kris' Blog about Otto</title>
```

Comment:
Very simple and perfect. Minimum viable product and checks every box. A good way to showcase skills. I think this would be a great example for others.